plugins/unicodelib/unicodelib.cpp

```diff
@@ -718,6 +719,46 @@ unsigned unicodeEditDistanceV4(UnicodeString & left, UnicodeString & right, unsi
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+void excludeNthWord(RuleBasedBreakIterator& bi, UnicodeString & source, unsigned n)
+{
+    bi.setText(source);
+    int32_t idx = bi.first();
+    int32_t wordidx = 0;
+    unsigned wordBeginning;
```

Comment:
trivial.  It would be worth assigning 0 to wordBeginning.  The value will never actually be used, but it will avoid some false positives from programs like Coverity which check the correctness of the program - since there is a path through the function where wordBeginning is not initialised (even though the path can never be taken).